RetailCoder.VBE/UI/Command/MenuItems/CodeMetricsCommandMenuItem.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Rubberduck.Parsing.VBA;
+using Rubberduck.UI.Command.MenuItems.ParentMenus;
+
+namespace Rubberduck.UI.Command.MenuItems
+{
+    public class CodeMetricsCommandMenuItem : CommandMenuItemBase
+    {
+        public CodeMetricsCommandMenuItem(CommandBase command) 
+            : base(command)
+        {
+        }
+        public override bool EvaluateCanExecute(RubberduckParserState state) => true;
```

Comment:
Pretty sure `true` is the default state.

Reply:
it isn't. The default state is `state != null && (_command?.CanExecute(state) ?? false);`. Since that's all irrelevant for showing a docked window it's overriden here